README.md

```diff
@@ -237,20 +244,21 @@ manual wrappers applied, the final generated bindings are placed under
 `pkg/nvml`.
 
 In general, the code used to generate the NVML Go bindings can be broken into 4 logical parts:
+
 1. Code defining the NVML API and how any auto-generated bindings should be produced
-1. Code responsible for dynamically loading `libnvidia-ml.so` on a host system and hooking it up to the bindings
-1. Code bridging the gap between any auto-generated bindings and the manual wrappers around them
-1. The manual wrappers themselves
-1. Test code
+2. Code responsible for dynamically loading `libnvidia-ml.so` on a host system and hooking it up to the bindings
+3. Code bridging the gap between any auto-generated bindings and the manual wrappers around them
+4. The manual wrappers themselves
+5. Test code
 
 Each of these parts is discussed in detail below, along with the files associated with them.
 
 ### Code defining the NVML API
 
 The following files aid in defining the NVML API and how any auto-generated bindings should be produced from it.
 
-* `gen/nvml/nvml.h`
-* `gen/nvml/nvml.yml`
+- `gen/nvml/nvml.h`
+- `gen/nvml/nvml.yml`
```

Comment:
I'd prefer to leave these as asterix. They render the same, but visually in the raw text, I think the asterix is nicer.

Reply:
`markdownlint` suggested me use dashes. I will revert it if asterix is prefered